float64 Left_tick_count
float64 right_tick_count

